daisy_workflows/image_build/debian/build_fai.py

```diff
@@ -134,6 +203,11 @@ def main():
 
   if returncode != 0:
     raise subprocess.CalledProcessError(returncode, cmd)
+  if run_sbom_bool:
+    fdisk_output = subprocess.run(['fdisk', '-l', work_dir + '/' + disk_name],
```

Comment:
what if fdisk fails?

Reply:
changed this to use sfdisk json, and catch errors in the command